sources/openwrt-http.go

```diff
@@ -31,9 +31,9 @@ func (s *openwrt) Run() error {
 	case "x86_64":
 		architecturePath = strings.Replace(s.definition.Image.ArchitectureMapped, "_", "/", 1)
 	case "armv7l":
-		architecturePath = "armvirt/32"
+		architecturePath = "armsr/armv7"
 	case "aarch64":
-		architecturePath = "armvirt/64"
+		architecturePath = "armsr/armv8"
```

Comment:
The `armvirt` part seems to still be used in releases and `armsr` appears to be for snapshots only? I don't know why they are using such a different layout though.

https://downloads.openwrt.org/releases/23.05.0-rc1/targets/armvirt/
https://downloads.openwrt.org/releases/22.03.5/targets/armvirt/
https://downloads.openwrt.org/snapshots/targets/armsr/

Reply:
Fixed